### ISSUE TITLE
[IoT] error thrown for azure-mgmt-iothub 0.2.0

### ISSUE THREAD

**Issue by xscript** [Azure/azure-sdk-for-python#920]
@lmazuel I am using azure-mgmt-iothub 0.2.0. Below error is thrown when calling the `list_event_hub_consumer_groups` API.
```
module 'azure.mgmt.iothub.models' has no attribute 'strPaged'
  ...
  File "...\env\lib\site-packages\azure\mgmt\iothub\operations\iot_hub_resource_operations.py", line 624, in list_event_hub_consumer_groups
    deserialized = models.strPaged(internal_paging, self._deserialize.dependencies)
AttributeError: module 'azure.mgmt.iothub.models' has no attribute 'strPaged'
```
I guess the root cause is the generated class is capitalized as `StrPaged`. But the one used in `list_event_hub_consumer_groups` is `strPaged`.
Please help fix this. Thanks!

**Comment by lmazuel** [comment]
Hi @xscript !
Thank you for your report! I take a look asap (today likely).

FYI @annatisch

**Comment by lmazuel** [comment]
Hi @xscript 

Released 0.2.1 with the fixes and added tests to avoid regression.

Close this issue, feel free to re-open it if necessary.
Thanks!

**Comment by xscript** [comment]
Thanks @lmazuel for the quick fix! 👍 
I will use the new package.
